src/runtime/ui-slot-composer.ts

```diff
@@ -31,10 +31,10 @@ export type SlotComposerOptions = {
 };
 
 export class UiSlotComposer {
-  private readonly _containerKind: string;
+  readonly _containerKind: string;
   readonly modality: Modality;
   readonly modalityHandler: ModalityHandler;
-  private readonly _consumers: SlotConsumer[] = [];
+  readonly _consumers: SlotConsumer[] = [];
```

Comment:
Can you get rid of the underscores? Our style now (such as we has one) is not to use underscores at all, but our former style would suggest that underscore means private. If they are no longer private then they definitely shouldn't have underscores.

Reply:
All of that stuff is tech-debt that comes from the original `slot-composer.ts` (that I forked). 

I continue to work on cleaning all that up, but it's going to take a while. At least, I'm pretty sure it's out of scope for this PR.

Reply:
If it's not actually needed, then reverting this file would be even better